src/tests/ConfigureRecordingSetTests.test.tsx

```diff
@@ -0,0 +1,176 @@
+/**
```

Comment:
Move the file next to the source file, and name it just `<source-file-name>.test.tsx`

Reply:
But yes, thanks for adding new tests!